Program.cs

```diff
@@ -146,44 +122,33 @@
 
 var app = builder.Build();
 
-// Configure the HTTP request pipeline.
-if (!app.Environment.IsDevelopment())
+if (app.Environment.IsProduction())
 {
     app.UseExceptionHandler("/error");
-    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
     app.UseHsts();
 }
-else
+else if (app.Environment.IsDevelopment())
 {
     app.UseDeveloperExceptionPage();
 }
 
-if (isProd == "false")
+if (app.Environment.IsDevelopment())
 {
-    Console.WriteLine("WE ARE IN DEVELOPMENT!");
     app.UseSwagger();
-    app.UseSwaggerUI(c =>
-    {
-        c.SwaggerEndpoint("/swagger/v1/swagger.json", "Chefster Backend");
-    });
+    app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "Chefster Backend"));
 }
 
 app.UseHangfireDashboard();
 app.MapHangfireDashboard();
 
-if (isProd == "true")
+if (app.Environment.IsProduction())
 {
     app.UseHttpsRedirection();
 }
```

Comment:
We could combine these together, not a big deal though 👍🏻

Reply:
I was thinking about that but the order that we add things to the `builder` and `app` matters so it might just be easiest to leave these environment checks as they currently are